__tests__/fixtures.ts

```diff
@@ -7,10 +7,32 @@ export const testEmployee: Employee = {
     email: "test@test.test",
 }
 
+export const testCustomer: Customer = {
+    id: 1,
+    name: "Some test customer",
+    description: "Used in tests.",
+}
+
+export const testProjectAllFields: Project = {
+    name: "Some project",
+    description: "Some description",
+    startDate: "2022-02-08",
+    endDate: "2022-02-09",
+    customer: testCustomer,
+    managingEmployee: testEmployee,
+}
+
+export const testProjectRequiredFields: Project = {
+    name: "Some project",
+    startDate: "2022-02-08",
+    customer: testCustomer,
+    managingEmployee: testEmployee,
```

Comment:
Perhaps we should add another employee fixture and set these testProjects to have different employees?

Reply:
I went on and created another test customer and employee. Might as well have different choices for both of those.